src/Action/RetrieveAutocompleteItemsAction.php

```diff
@@ -213,6 +213,11 @@ private function retrieveFormFieldDescription(
         AdminInterface $admin,
         string $field
     ): FieldDescriptionInterface {
+        /**
+         * NEXT_MAJOR: remove annotation. Return type of interface will be correct.
+         *
+         * @var FieldDescriptionInterface|null
```

Comment:
shouldn't you change this on AdminInterface phpdoc?

Reply:
Probably yes, I thought that non nullable is more strict

Reply:
What is the typehint on the method?

Reply:
@jordisala1991 it was just `FieldDescriptionInterface` I've changed it to `FieldDescriptionInterface|null`